### PR TITLE
[WIP] 3772: Add loan tab view

### PR DESCRIPTION
**Pull request by emitche**

### CODE REVIEW

File: app/assets/javascripts/backbone/views/loan_tab_view.coffee [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class MS.Views.LoanTabView extends Backbone.View
+
+  events: (params) ->
```
Comment (hooverlunch) [inline app/assets/javascripts/backbone/views/loan_tab_view.coffee]: events should be a hash, not a function. So remove `(params) ->` from here and your event handler should start working.

File: app/assets/javascripts/backbone/views/loan_tab_view.coffee [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    'click .calendar-tab': 'openCalendar'
+
+  openCalendar: () ->
```
Comment (hooverlunch) [inline app/assets/javascripts/backbone/views/loan_tab_view.coffee]: You can remove `()`

File: app/views/admin/loans/show.html.slim [hunk 3]
```diff
@@ -36,3 +36,8 @@ section.loan.details class=(@loan.valid? ? 'show-view' : 'edit-view')
 
   div.loan-content
     = render "admin/loans/tabs/tabs"
+
+javascript:
+  $(function(){
+    new MS.Views.LoanTabView();
```
Comment (hooverlunch) [inline app/views/admin/loans/show.html.slim]: You need to set `el` or Backbone won't work. So `new MS.Views.LoanTabView({ el: 'something' });`

**Comment by emitche** [comment]
Thanks for the help!

**Pull request merged**